code/modules/mob/living/brain/brain_item.dm

```diff
@@ -262,7 +262,11 @@
 	if(prob(25))
 		return
 
-	owner.adjustOrganLoss(ORGAN_SLOT_BRAIN, 50/severity)
+	var/obj/item/clothing/head/foilhat/hat = owner.get_item_by_slot(SLOT_HEAD)
```

Comment:
type stuffing is GROSS

Reply:
what does this mean

Reply:
probably that you're assigning the hat as a foilhat no matter what the hat actually is

Reply:
i mean there's no functional difference between doing this and using istype() and just assigning it as a normal hat and then checking if it's the foilhat type right?

Reply:
changed it i guess